### PR TITLE
Add ability to remove or send a log files to developer

### PR DESCRIPTION
**Pull request by ray1997**

### CODE REVIEW

File: Quick Pad/Dialog/Settings.xaml.cs [hunk 1]
```diff
@@ -374,6 +376,46 @@ public async void LoadCrashList()
...
+
+        public async void SendCrashReports()
+        {
+            EmailMessage emailMessage = new EmailMessage()
+            {
+                Body = "[QuickPad][CrashReport] I want to send these crash report",
+            };
+            string front = "something_???";
+            string provider = "outlook?";
+            emailMessage.To.Add(new EmailRecipient($"{front}@{provider}.com"));
```
Comment (ray1997) [inline Quick Pad/Dialog/Settings.xaml.cs]: Still, don't know which email to send to.
Please don't put the email directly or some bot that crawling on the internet might put you on spam list :/

Comment (yaichenbaum) [reply Quick Pad/Dialog/Settings.xaml.cs]: I dont need it to send directly, more like an option for the user to manually send if they want help.

Comment (ray1997) [reply Quick Pad/Dialog/Settings.xaml.cs]: Maybe just copy a text will do?

File: Quick Pad/Dialog/Settings.xaml.cs [hunk 2]
```diff
@@ -374,6 +374,46 @@ public async void LoadCrashList()
...
+                AllCrashes = new ObservableCollection<CrashInfo>();
+            else
+                AllCrashes.Clear();
+        }
+
+        public async void SendCrashReports()
+        {
+            EmailMessage emailMessage = new EmailMessage()
+            {
+                Body = "[QuickPad][CrashReport] I want to send these crash report",
```
Comment (yaichenbaum) [inline Quick Pad/Dialog/Settings.xaml.cs]: body should be "[Quick Pad][Crash Report] {Version Number},

File: Quick Pad/Dialog/Settings.xaml [hunk 3]
```diff
@@ -248,7 +248,11 @@
                                 <Run Text="Give yourself a chance to save or copy your work."/>
                                 <LineBreak/><Run Text="You may need to restart as some functions may not work properly."/>
                         </TextBlock>
-
+                        
+                        <StackPanel Visibility="{x:Bind ShowCrashList, Mode=OneWay}" Orientation="Horizontal">
+                            <Button Content="Clear crash list" Click="{x:Bind ClearCrashList}"/>
```
Comment (yaichenbaum) [inline Quick Pad/Dialog/Settings.xaml]: Content="Clear crash history"

**Pull request merged**